src/app/series/directives/series-feed.component.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { DistributionModel, SeriesModel, StoryModel, TabService } from '../../shared';
+
+@Component({
+  template: `
+    <section *ngIf="series && podcast">
+      <div class="hint" *ngIf="noStories">
+        You have no published stories in this podcast.
+      </div>
+
+      <section *ngIf="!noStories">
+        <ul *ngIf="privateStories.length">
+          <h4>Private</h4>
+          <li *ngFor="let s of privateStories">
+            <h5><a [routerLink]="['/story', s.id]">{{s.title}}</a></h5>
+            <p class="private">No publication date set.</p>
+          </li>
+        </ul>
+
+        <ul *ngIf="futurePublicStories.length">
+          <h4>Set for future publication</h4>
+          <li *ngFor="let s of futurePublicStories">
+            <h5><a [routerLink]="['/story', s.id]">{{s.title}}</a></h5>
+            <p class="futurePublic">Will be published {{s.pubDate}}.</p>
+          </li>
+        </ul>
+
+        <ul *ngIf="publicStories.length">
+          <h4>Public feed</h4>
+          <li *ngFor="let s of publicStories">
+            <h5><a [routerLink]="['/story', s.id]">{{s.title}}</a></h5>
+            <p class="public">Published {{s.pubDate}}.</p>
+          </li>
+        </ul>
+      </section>
+    </section>
+  `,
+  styleUrls: ['./series-feed.component.css']
+})
+
+export class SeriesFeedComponent implements OnDestroy {
+
+  noStories: boolean;
+  series: SeriesModel;
+  publicStories: StoryModel[] = [];
+  futurePublicStories: StoryModel[] = [];
+  privateStories: StoryModel[] = [];
+  podcast: DistributionModel;
+  tabSub: Subscription;
+
+  constructor(tab: TabService) {
+    this.noStories = false;
+    this.tabSub = tab.model.subscribe((s: SeriesModel) => {
+      this.series = s;
+      this.podcast = s.distributions.filter(dist => dist.kind === 'podcast')[0];
+      this.sortStories();
+    });
+  }
+
+  sortStories() {
+    this.podcast
+        .parent
+        .followItems('prx:stories', { sorts: 'updated_at:desc' })
```

Comment:
Agreed. One other odd thing is -- I think the `total` and `count` number of stories in the _embedded may only be those that are currently published, while _links is showing all stories regardless of pub status... ? Could be coincidence but I have a series of 12 stories, and here's what I'm seeing: 
```javascript
this.series.doc._embedded['prx:stories'].count //= 4
this.series.doc._embedded['prx:stories'].total //= 4
this.series.doc._links['prx:stories'].count //= 12
```

Could it be because the _embedded uses the non-authorized uri
`href: "/api/v1/series/36512/stories"`
and _links uses the authorized one `href: "/api/v1/authorization/series/36512/stories{?page,per,zoom,filters,sorts}"`
?